app.rb

```diff
@@ -245,7 +245,7 @@ def convert_relpath_to_url(relpath)
     # This works around a bug in S3:
     # https://forums.aws.amazon.com/message.jspa?messageID=207700
     relpath.gsub!(/\+/, "%2B")
-    base = "#{request.scheme}://#{channel.aws_packages_bucket}.s3.amazonaws.com"
+    base = "https://#{channel.aws_packages_bucket}.s3.amazonaws.com"
```

Comment:
We should add a comment above this change that we'll switch things back once ECO-121 is resolved.